fdbserver/storageserver.actor.cpp

```diff
@@ -10848,26 +10852,28 @@ void changeServerKeysWithPhysicalShards(StorageServer* data,
 		}
 		if (!currentShard.isValid()) {
 			if (currentRange != keys) {
-				TraceEvent(SevError, "PhysicalShardStateError")
+				TraceEvent(sev, "PhysicalShardStateError")
 				    .detail("SubError", "RangeDifferent")
 				    .detail("CurrentRange", currentRange)
 				    .detail("ModifiedRange", keys)
 				    .detail("Assigned", nowAssigned)
 				    .detail("DataMoveId", dataMoveId)
-				    .detail("Version", version);
-				throw internal_error();
+				    .detail("Version", version)
+				    .detail("InitialVersion", currentShard->version);
+				throw data_move_conflict();
```

Comment:
Is there any difference when SS handles internal_error and data_move_conflict?

Reply:
No. They all fail the storage error.